src/public-modules/Fulfillment/sagas.js

```diff
@@ -16,8 +51,104 @@ export function* loadFulfillment(action) {
   }
 }
 
+export function* createFulfillment(action) {
+  const { bountyId, data } = action;
+  const {
+    name,
+    email,
+    url,
+    description,
+    fileName,
+    ipfsHash: fulfillmentDataHash
+  } = data;
+
+  yield put(setPendingWalletConfirm());
+
+  const userAddress = yield select(addressSelector);
+  const { web3 } = yield call(getWeb3Client);
+
+  const payload = {
+    payload: {
+      url,
+      description,
+      sourceFileName: fileName,
+      sourceDirectoryHash: fulfillmentDataHash,
+      sourceFileHash: '',
+      fulfiller: {
+        email,
+        userAddress,
+        name
+      }
+    },
+    meta: {
+      platform: 'bounties-network',
+      schemaVersion: '0.1',
```

Comment:
Is this version something we should set as a constant somewhere?

Reply:
definitely